test/integration/master/transformation_testcase.go

```diff
@@ -86,11 +85,13 @@ func newTransformTest(l kubeapiservertesting.Logger, transformerConfigYAML strin
 		return nil, fmt.Errorf("error while creating rest client: %v", err)
 	}
 
-	if e.ns, err = e.createNamespace(testNamespace); err != nil {
+	testNamespaceName := string(uuid.NewUUID())
```

Comment:
keep the names `testNamespace` and `testSecret`

Reply:
I can access these names via ns.Name and secret.Name after those objects are created.